### DISCUSSION TITLE
WebUi ne se charge pas...

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#1635] [Q&A]
Bonsoir, 

je viens de télécharger le script ce soir 26/02 et tout a l'air de fonctionner, je n'ia aucun message d'erreur mais 
au final quand je clique sur le lien WebUi ne se télécharge par et j'ai un message 504 Gateway Time-out 

des idées ?

Merci d'avance

**Comment by TheLastBen** [comment]
Utilise localtunnel pour le moment, c'est probablement un problème des serveurs gradio
